redis/deprecated/redisbp/monitored_client.go

```diff
@@ -2,12 +2,19 @@ package redisbp
 
 import (
 	"context"
+	"errors"
+	"fmt"
 	"time"
 
 	"github.com/go-redis/redis/v8"
+
 	"github.com/reddit/baseplate.go/metricsbp"
 )
 
+// ErrReplicationFactorFailed returns when the cluster client wait function returns replica reached count
+// that is less than desired replication factor
+var ErrReplicationFactorFailed = errors.New("redis: failed to meet the requested replication factor")
```

Comment:
nit: this should be `redisbp:` instead of `redis:` (use the name of the package for the error prefix).